README.md

```diff
@@ -22,7 +22,65 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+### Scraping
+
+`NewsScraper::Scraper#scrape` will allow either yields the transformed data or returns an array of the transformed data for all Google News RSS articles for a query.
```

Comment:
`NewsScraper::Scraper#scrape` will either yield the transformed data or returns an array of the transformed data for all Google News RSS articles for a query.` is better english.

Doesn't it technically return the array either way?